src/harness/testcases/WINNF_FT_S_DRG_testcase.py

```diff
@@ -269,8 +272,175 @@ def test_WINNF_FT_S_DRG_5(self):
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Load grant.json
+    grant_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+
+    # Heartbeat requests (where testcode will insert cbsdId and grantId)
+    heartbeat_a = {'operationState': 'GRANTED'}
+    heartbeat_c = {'operationState': 'GRANTED'}
+    heartbeat_b = {'operationState': 'GRANTED'}
+
+    # Deregistration requests (where testcode will fill in cbsdId)
+    # device_a's CBSD ID will be the same test string.
+    deregister_a = {'cbsdId': 'Test-Invalid-cbsd-string'}
+    # device_c's CBSD ID will be removed due to the 'REMOVE' keyword.
+    deregister_c = {'cbsdId': 'REMOVE'}
+    deregister_b = {}
+
+    # Device_a and device_c are Category A.
+    self.assertEqual(device_a['cbsdCategory'], 'A')
+    self.assertEqual(device_c['cbsdCategory'], 'A')
+
+    # Device_b is Category B with conditionals pre-loaded.
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+    conditionals_b = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam']
+    }
+    conditionals = {'registrationData': [conditionals_b]}
+    del device_b['installationParam']
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+
+    # Create the actual config.
+    devices = [device_a, device_c, device_b]
+    grant_requests = [grant_a, grant_c, grant_b]
+    heartbeat_requests = [heartbeat_a, heartbeat_c, heartbeat_b]
+    deregistration_requests = [deregister_a, deregister_c, deregister_b]
+    config = {
+        'registrationRequest': devices,
+        'conditionalRegistrationData': conditionals,
+        'grantRequest': grant_requests,
+        'heartbeatRequest': heartbeat_requests,
+        'deregistrationRequest': deregistration_requests,
+        # First request has an invalid CBSD ID => INVALID_VALUE
+        # Second request is missing CBSD ID => MISSING_PARAM
+        # Third request, all valid params => SUCCESS
+        'expectedResponseCodes': [(103,), (102,), (0,)]
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_DRG_6_default_config)
+  def test_WINNF_FT_S_DRG_6(self, config_filename):
+    """[Configurable] Array Deregistration."""
+
+    config = loadConfig(config_filename)
+    # Very light checking of the config file.
+    self.assertEqual(
+        len(config['registrationRequest']), len(config['grantRequest']))
+    self.assertTrue(
```

Comment:
should this be assertEqual   instead of assertTrue ?

Reply:
Fixed.